### PR TITLE
:question: Add open and closed results toggle

### PR DESCRIPTION
**Pull request by AdamChrimes**
## Description
Add an on & off toggle to filter the results by open & closed pharmacies.

## Motivation and Context

Reduce the number of people not finding open pharmacy information on evenings and weekends

## Checklist

- [ ] An [ADR](http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions)
- [ ] New and/or updated tests
- [x] Changes log in `CHANGELOG`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,14 @@
+0.35.0 / TBA
```
Comment (st3v3nhunt) [inline CHANGELOG.md]: No need for a new version (the previous one hasn't been released yet), just add the change to `0.34.0`

File: scss-c2s/c2s-results.scss [hunk 2]
```diff
@@ -403,3 +443,5 @@ body {
     }
   }
 }
+
```
Comment (st3v3nhunt) [inline scss-c2s/c2s-results.scss]: Can this blank line be removed please?

File: scss-c2s/c2s-results.scss [hunk 3]
```diff
@@ -403,3 +443,5 @@ body {
...
+
```
Comment (st3v3nhunt) [inline scss-c2s/c2s-results.scss]: Can this blank line be removed please?

**Comment by AdamChrimes** [comment]
Fixed the version numbers & white space

**Comment by st3v3nhunt** [comment]
This comment seems to have disappeared...
> The version number in `package.json` needs reverting too please.

**Pull request merged**